### PR TITLE
feat(custom_filename): custom output filename

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Context 

Add flag `-F` to put a custom filename for Prowler output reports.

### Description

Add flag `-F` to put a custom filename for Prowler output reports.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: lib/outputs/outputs.py [hunk 1]
```diff
@@ -101,13 +105,16 @@ def report(check_findings, output_options, audit_info):
...
 
 
-def fill_file_descriptors(output_modes, audited_account, output_directory, csv_fields):
+def fill_file_descriptors(
+    output_modes, audited_account, output_directory, csv_fields, output_filename
+):
     file_descriptors = {}
+    # If no input custom output filename, set default:
+    if not output_filename:
+        output_filename = f"prowler-output-{audited_account}-{output_file_timestamp}"
```
Comment (jfagoagas) [inline lib/outputs/outputs.py]: You are doing the same at lines 250 and 251, why don't you include this at a higher level so both functions always receive a non-empty `output_filename`. What do you think?

File: prowler [hunk 2]
```diff
@@ -132,9 +132,16 @@ if __name__ == "__main__":
         help="Output mode, by default csv",
         choices=["csv", "json", "json-asff"],
     )
+    parser.add_argument(
+        "-F",
+        "--output-filename",
+        nargs="?",
+        default=None,
+        help="Custom output report name, if not specified will use default output/prowler-output-ACCOUNT_NUM-OUTPUT_DATE.format.",
+    )
```
Comment (jfagoagas) [inline prowler:141]: 👏 👏 👏 👏

**Review by jfagoagas — CHANGES_REQUESTED** [review]

File: lib/outputs/outputs.py (hunk 1 above)
Comment (MrCloudSec) [reply lib/outputs/outputs.py]: Great idea! I put it at a high level now.

**Review by jfagoagas — APPROVED** [review]
Good job @sergargar it looks great!!!!

**Pull request merged**